new/LIT7275QeneWaEsagged.xml

```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7275QeneWaEsagged" xml:lang="en" type="work">
+            <teiHeader>
+                <fileDesc>
+                    <titleStmt>
+                        <title xml:lang="gez" xml:id="t1">እሰግድ፡ በብረክየ፡ ወበልብየ፡ እገኒ፡ በዕፀ፡ ቅዱስ፡ መስቀል።</title>
+                        <title xml:lang="gez" type="normalized" corresp="#t1">ʾƎsaggǝd ba-bǝrakǝya wa-ba-lǝbbəya ʾǝgani ba-ʿǝḍa qǝddus masqal ... 
+                            (qǝne of the wāzemā-type)</title>
+                        <title xml:lang="en" corresp="#t1">I prostrate on my knees and in my heart I praise by the wood of the holy cross ...</title>
+                        <editor role="generalEditor" key="AB"/>
+                        <editor key="CH"/>
+                        <funder>Akademie der Wissenschaften in Hamburg</funder>
+                    </titleStmt>
+                    <publicationStmt>
+                        <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                        <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                        <pubPlace>Hamburg</pubPlace>
+                        <availability>
+                            <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                        </availability>
+                    </publicationStmt>
+                    <sourceDesc>
+                        <p/>
+                    </sourceDesc>
+                </fileDesc>
+                <encodingDesc>
+                    <p>A digital born TEI file</p>
+                </encodingDesc>
+                <profileDesc>
+                    <creation/>
+                    <abstract>
+                        <p>Neither type nor author of the qǝne are indicated in the manuscript. According to the classification by 
```

Comment:
If you agree that the ዋ most probably stands for ዋዜማ, then the qene type is indeed indicated